src/NotesField.php

```diff
@@ -10,14 +10,66 @@ class NotesField extends Field
     public $showOnCreation = false;
     public $showOnIndex = false;
 
+    public $showAddNote = true;
+
+    /**
+     * NovaNotesField constructor.
+     *
+     * @param $name
+     * @param null $attribute
+     * @param callable|null $resolveCallback
+     */
+    public function __construct($name, $attribute = null, callable $resolveCallback = null)
+    {
+        parent::__construct($name, $attribute, $resolveCallback);
+
+        $this->withMeta([
+            'showAddNote' => $this->showAddNote,
+        ]);
+    }
+
     /**
      * Sets the placeholder value displayed on the field.
      *
      * @param string $placeholder
-     * @return \OptimistDigital\NovaNotesField\NotesField
+     * @return NotesField
      **/
     public function placeholder($placeholder)
     {
         return $this->withMeta(['placeholder' => $placeholder]);
     }
+
+    /**
+     * Sets the input size.
+     *
+     * @param string $inputWidth
+     * @return NotesField
+     **/
+    public function inputWidth($inputWidth)
+    {
+        return $this->withMeta(['inputWidth' => $inputWidth]);
+    }
+
+    /**
+     * Show or hide the AddNote input.
+     *
+     * @param string $showAddNote
+     * @return NotesField
+     **/
+    public function showAddNote($showAddNote)
```

Comment:
Can this be renamed to showAddNoteButton please?

Reply:
No problem, I'll also update the readme to reflect this.